README.md

```diff
@@ -11,7 +11,7 @@ Factory Muffin 3.0
 
 The goal of this package is to enable the rapid creation of objects for the purpose of testing.
 
-It's basically a "[factory\_girl](https://github.com/thoughtbot/factory_girl)", simplified for use with PHP.
+It's basically a "[factory\_bot](https://github.com/thoughtbot/factory_bot)", simplified for use with PHP.
```

Comment:
Maybe change this to
```markdown
It's basically a "[factory\_bot](https://github.com/thoughtbot/factory_bot)" (formerly factory_girl), simplified for use with PHP
```
?

I'll admit: when I read the readme my first thought was, "didn't they rename that lib to something else?"

Reply:
Technically, this package was not based on factory bot though, it was based on "factory girl", as it stood then.